### ISSUE TITLE
Views replacement patterns

### ISSUE THREAD

**Issue by soniktrooth** [drupalprojects/kalatheme#200]
![screen shot 2015-09-01 at 12 00 08 pm](https://cloud.githubusercontent.com/assets/597617/9614438/a6eccd20-50a6-11e5-8e29-b88f870a74c3.png)

I've seen this on a few projects now. 

It has something to do with the theme function for textareas being different from the theme functions of other form inputs.

In kalatheme_process_input() we are adding `form-control` class to the #attributes array which seems to get put on the input in most cases but for textarea it seems to go on the wrapper which then breaks things.

I have implemented a css only fix on HSU which looks a little something like this:

```
#edit-options .collapsible {
  float: none;
  width: 95%;
}

#edit-options-alter-text-wrapper {
  height: auto;
  width: 95%;
```

Whilst this works, I do wonder if a more appropriate placing of the `form-control` class on textareas would be a better solution.

**Comment by RobLoach** [comment]
Is it a conflict the CSS? Some CSS that we're adding that conflicts with the Views UI?

**Comment by soniktrooth** [comment]
@RobLoach no, I think the styling for `form-control` is from bootstrap and as part of the BS integration we're adding the class to various form inputs. In the case of textareas the class is getting the class on a wrapper div instead of on the textarea element.
